chromium_src/chrome/browser/resources/extensions/detail_view.ts

```diff
@@ -3,12 +3,9 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // you can obtain one at https://mozilla.org/MPL/2.0/.
 
-// @ts-nocheck TODO(petemill): Define types and remove ts-nocheck
-
 import {RegisterPolymerTemplateModifications} from 'chrome://resources/brave/polymer_overriding.js'
 import {loadTimeData} from 'chrome://resources/js/load_time_data.js';
 
-
 RegisterPolymerTemplateModifications({
```

Comment:
This override doesn't apply and hasn't since the upstream component was converted to Lit. I'll fix this in a followup
https://github.com/brave/brave-browser/issues/42691